napari/utils/colormaps/colormap.py

```diff
@@ -0,0 +1,73 @@
+from enum import auto
+
+import numpy as np
+
+from ...utils.misc import StringEnum
+from .colorbars import make_colorbar
+from .standardize_color import transform_color
+
+
+class ColormapInterpolationMode(StringEnum):
+    """INTERPOLATION: Interpolation mode for colormaps."""
```

Comment:
I believe we include a definition of the enum values in our other mode enums. I think it would be nice to do so here as well.